docs/package-imports.md

```diff
@@ -0,0 +1,41 @@
+An agent that is generated from the AEA framework is a modular system with different skills, connections, and protocols.
+When we create a new agent with the command `aea create my_aea` we create the file structure that looks like the following :
+- connections
+- protocols
+- skills
+- vendor
+- aea-config.yaml
+
+In all the folders except the `vendor`, we can locate the packages we developed. On the other hand, if we download a 
+package from the registry it will be located inside the `vendor's folder` that follows a similar structure.
+
+To use a package, the path for the package must be listed inside the `aea-config.yaml` file.
+```yaml
+connections:
+- fetchai/stub:0.1.0
+```
+The above shows a part of the `aea-config.yaml`. If you see the connections, you will see that we follow a pattern of `author/name_package: version` to identify each package. The author that we are using is the author of the package and not 
+the author that shows the `aea-config.yaml` file. This indicates, that we are using a package that is made by Fetch.ai and is located inside the `vendor/fetchai/connections` folder.
+
+The way we import packages inside the agent is in the form of `packages.author.package_type.package_name.module`. This happens
```

Comment:
The way we import packages inside the agent is in the form of `packages.author.package_type.package_name.module_name`. So for the above example, the import path is `packages.fetchai.connections.stub.module_name`.

Reply:
The framework loads the modules from the local agent project and adds them to Python's `sys.modules` under the respective path.